### ISSUE TITLE
[BUG] "Delete tag" elements are no button/anchor

### ISSUE THREAD

**Issue by wnhrt** [paperless-ngx/paperless-ngx#6783]
### Description

The tag elements in the detail editor of a document, with which added tags can be displayed and deleted, are neither buttons nor anchors.

![image](https://github.com/paperless-ngx/paperless-ngx/assets/70302993/ec3ad451-692a-4020-8f76-e2f97ee71a54)

This restricts accessibility, i.e. accessibility tools cannot recognize these elements. As they are clickable and offer functionality, they should be of an appropriate type.

(Tags inside the "Tags" column of the dashboard are also affected, but that might be different enough to justify a separate issue.)

### Steps to reproduce

1. Open document with tag or add tag in detail view of new document
2. Inspect element, e.g. with browser debug tool

### Webserver logs

```bash
-
```

### Paperless-ngx version

2.8.6

### Host OS

Debian

### Installation method

Docker - official image

### Browser

Firefox

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

**Comment by shamoon** [comment]
I appreciate that there are accessibility gains to be made but I’m going to convert this to a request, not a bug in the basic sense of the word. It has been like this for literally years so it’s not like a bug was introduced.

When I have a second I can look into it.
